CHANGES.rst

```diff
@@ -4,6 +4,18 @@ Latest changes
 Development
 -----------
 
+Thomas Moreau
+
+    Limit the number of threads used in worker processes (for ``loky`` and
+    ``multiprocessing``) by C-libraries that relies on threadpools. This
+    functionality works for MKL, OpenBLAS, OpenMP and Accelerated.
+
+    Implement the ``loky`` backend with @ogrisel. This backend relies on
+    a robust implementation of ``concurrent.futures.ProcessPoolExecutor``
+    with spawned processes that can be reused accross the ``Parallel``
+    calls.
```

Comment:
Please move this to the top and explain that it fixes bad interactions (crash) with the internal thread-pool of third-party libraries with a link to.

https://pythonhosted.org/joblib/parallel.html#bad-interaction-of-multiprocessing-and-third-party-libraries